res/script.js

```diff
@@ -14,6 +14,13 @@ window.onload = function () {
         $('details[data-level=2]').attr('open', false);
     }
 
+    if (window.innerWidth < 720 && $('details')) {
+        var HighestLevel = Math.min(...$('details').map( function() {return $(this).attr('data-level');}).get());
+        if($(`details[data-level=${HighestLevel}]`).length == 1){
+            $(`details[data-level=${HighestLevel}]`).attr('open', true);
+        }
+    }
+
```

Comment:
The approach is probably OK, but I believe we could do better:
* Put everything under the `if (window.innerWidth < 720)` test to avoid duplicate test
* Put comments for each scenario ( we have two scnearios here)
* `$('details[data-level=2]')` is wrong as well, as we need to apply to the toplevel sections (could be different level than 2).
* We should probably have: if only one top section then do nothing (otherwise open everything).

Reply:
ok